dev/Pager/InteractionTests/PagerTests.cs

```diff
@@ -812,6 +823,19 @@ void VerifyDisplayMode(DisplayModes mode)
             switch (mode)
             {
                 case DisplayModes.Auto:
+                    if (elements.GetNumberOfPagesTextBlock().GetText() == "100")
```

Comment:
I think it might be better to make this a bit more flexible, e.g. try to convert this to int and check if its higher than lets say 50.

Reply:
Or ideally use the magic const 11 we use in the actual control.